zeebe/gateway-protocol/src/main/proto/v2/search-models.yaml

```diff
@@ -6,7 +6,10 @@ components:
       type: object
       properties:
         page:
-          $ref: '#/components/schemas/SearchQueryPageRequest'
+          description: Pagination criteria.
+          allOf:
+            - $ref: '#/components/schemas/SearchQueryPageRequest'
+          type: object
```

Comment:
🔧 We can completely leave this out, I would say. This can use all the description and stuff from the `$ref`.